extensions/cornerstone-dicom-seg/src/utils/hydrationUtils.ts

```diff
@@ -61,6 +64,14 @@ async function updateViewportsForSegmentationRendering({
     };
     const viewportId = viewport.viewportId;
 
+    // maintain the prehydration slice on the target viewport only
+    if (viewportId === targetViewportId) {
+      viewport.viewportOptions.initialImageOptions = {
+        index: initialSliceIndex,
+        useOnce: true,
```

Comment:
Why do we need `useOnce`? I don't prefer this approach.

Reply:
So when the seg is hydrated, the initial image index is set, but lets say the user switches to another series, then drops the referenced series for the seg again, we should not use the initial index again, we should default to first/middle or last.